plugins/auth-backend/src/providers/PassportStrategyHelper.ts

```diff
@@ -16,7 +16,43 @@
 
 import express from 'express';
 import passport from 'passport';
-import { RedirectInfo, RefreshTokenResponse } from './types';
+import jwtDecoder from 'jwt-decode';
+import { RedirectInfo, RefreshTokenResponse, ProfileInfo } from './types';
+
+export const makeProfileInfo = (
```

Comment:
Bit worried that this is gonna be a bit too different between providers, but let's hit that problem and deal with it later :grin: